<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProxyOptionsWidget</class>
 <widget class="QWidget" name="ProxyOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>311</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Proxy</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_21">
   <item>
    <widget class="QGroupBox" name="boxProxyServer">
     <property name="title">
      <string> Proxy Server  </string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_16">
      <item>
       <widget class="QLabel" name="lblProxyServer">
        <property name="text">
         <string>Server</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editProxyServer">
        <property name="text">
         <string>127.0.0.1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblProxyPort">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinProxyPort">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>80</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="boxProxyAuth">
     <property name="title">
      <string>Use proxy authentication  </string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_17">
      <item>
       <widget class="QLabel" name="ProxyUserNameLbl">
        <property name="text">
         <string>User name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ProxyUserEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="ProxyPasswordLbl">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ProxyPswdEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="boxProxyBypassList">
     <property name="title">
      <string>Proxy bypass list</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_20">
      <item>
       <widget class="QLabel" name="lblProxyBypassListHlp">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Use semicolon (;) to separate entries </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="plainProxyBypassList">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="plainText">
         <string>&lt;local&gt;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
